notebooks/en/rag_with_unstructured_data.ipynb

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
<p>"You can use the Unstructured library..."</p><p> </p><p>"...but feel free to explore other options in the Unstructured documentation."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>